resources/test/tests/unit/late-test.html

```diff
@@ -27,7 +28,7 @@
         "test(function() {}, 'acceptable test');",
         "onload = function() {",
         "  done();",
-        "  test(function() {}, 'this test is late and should be ignored');",
+        "  test(function() {}, 'this test is late and should NOT be ignored');",
```

Comment:
Since we accept this now it's not a late test, so let's spell out what the test is in the description:

```suggestion
        "  test(function() {}, 'test registered in onload handler');",
```